_templates/troubleshooting.md

```diff
@@ -0,0 +1,113 @@
+---
+title: [Title]
+description: [Brief description of the How-To article]
+type: [Troubleshooting]
+teampulseid: [number]
+ticketid: [number]
+pitsid: [number] <!-- Not sure we need this, as pits item are related to teampulse anyway -->
+<!-- TO DO: What else should we have here? -->
+---
+
+# Title
+<!-- Mandatory, Character Limit: 255 -->
+<!-- Recommendation is to have <= 75 characters for a title as this can impact indexing by search engines like Google, Bing, etc. -->
+<!-- The article title should be a combination of the main problem statement and if applicable, the key environment statement. -->
+
+
+
+<!-- Environment Table -->
+<!-- TO DO:
+     We need to check how such can be developed according to our needs:
+ 	 We don't need Header roll, but rather a title: Environment.
+     E.g. - Telerik Product, Version, SDK, SDK version, Mobile OS Vendor, Mobile OS Vendor Version, Client, Client Version, App Type (Hybrid/NativeScript/Native), 3rd party integration, Browser Type, Browser Version -->
+| Service:     | [AppBuilder], [Backend Services], [AppManager] |
+| Other stuff: | [Other stuff here]                             |
+
+
+
+## Description
+<!-- Mandatory -->
+State the question.
+
+* Use complete thoughts.
+* Use words and phrases the customer used to describe the issue (even if technically inaccurate).  
+* If possible restate the problem in technical wording. This increases the ability to find the article for internal staff.
+
+## Steps to Reproduce\Additional Details
+<!-- Optional -->
+<!-- TO DO: Decide which name suits better -->
```

Comment:
We can have additional details in each section (not only the steps to reproduce). This section will stay "Steps to Reproduce".